nixos/doc/manual/development/writing-nixos-tests.section.md

```diff
@@ -21,10 +21,13 @@ import ./make-test-python.nix {
 }
 ```
 
-The attribute `testScript` is a bit of Python code that executes the
+We refer to the whole test above as a test module, whereas the values
+in [`nodes.<name>`](#opt-nodes) are NixOS modules themselves.
```

Comment:
I think it makes more sense to not use the term "NixOS module" but "NixOS configuration" because NixOS configurations (as they are first seen by users in the `configuration.nix` are a more specific type of function than modules, they just make use of the module convention.